modules/oxide-components/src/main/ts/keynav/keyboard/flowtype/stories/FlowTypeDemo.tsx

```
@@ -0,0 +1,74 @@
+const defaultExecute = (
+  focused: SugarElement<HTMLElement>
+): Optional<boolean> => {
+  focused.dom.click();
+  return Optional.some(true);
+
+};
+
+export const FlowTypeDemo: React.FC<Omit<FlowKeyingProps, 'containerRef'>> = ({
+  selector,
+  execute = defaultExecute,
+  escape = Fun.constant(Optional.none()),
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)